### PR TITLE
Add facebook login

### PR DESCRIPTION
**Pull request by JessSaavedra**
## Summary

- Added Facebook Login

## Trello Card

https://trello.com/c/DOKZmvtd

### CODE REVIEW

File: config/initializers/devise.rb [hunk 1]
```diff
@@ -263,4 +263,5 @@
   # so you need to do it manually. For the users scope, it would be:
   # config.omniauth_path_prefix = '/my_engine/users/auth'
   config.omniauth :google_oauth2, ENV['GOOGLE_CLIENT_ID'], ENV['GOOGLE_CLIENT_SECRET']
+  config.omniauth :facebook, ENV['FACEBOOK-ID'], ENV['FACEBOOK-SECRET']
```
Comment (epintos) [inline config/initializers/devise.rb]: Same with credentials as the other PR

File: config/initializers/devise.rb [hunk 2]
```diff
@@ -262,5 +262,8 @@
   # When using OmniAuth, Devise cannot automatically set OmniAuth path,
   # so you need to do it manually. For the users scope, it would be:
   # config.omniauth_path_prefix = '/my_engine/users/auth'
-  config.omniauth :google_oauth2, ENV['GOOGLE_CLIENT_ID'], ENV['GOOGLE_CLIENT_SECRET']
+  login_credentials = AppConfiguration.for(:login)
```
Comment (epintos) [inline config/initializers/devise.rb]: You could have used secrets.yml here by having an account only for development. But its okay anyway

**Comment by epintos** [comment]
:+1:

**Comment by JessSaavedra** [comment]
Merged manually

**Pull request closed without merging**